source/process/regex-library.md

```diff
@@ -1,83 +1,84 @@
 ---
 title: RegEx Library
 ---
+
 <details close>
 <summary>General</summary>
 - **extract text**: in the Find window, choose <mark>'Extract'</mark> to pull contents from a file or project<br>F: <code>&#60;body(?msi)(.&#42;?)&#60;/body&#62;</code>
 
 - **extract classes**: choose <mark>'Extract'</mark> to pull classes from a file or project<br>F: <code>\sclass=&#34;\[^&#34;]+&#34;</code>
 
-- **remove divs**: Find divs and replace with only the div content<br>F: <code>&#60;div(?: class=&#34;\[^&#34;]+&#34;)?>((?:.|\s)&#42;?)&#60;/div&#62;</code><br>R: <code>\1</code>
+- **remove divs**: Find divs and replace with only the div content<br>F: <code>&#60;div(?: class=&#34;\[^&#34;]+&#34;)?>((?:.|\s)\*?)&#60;/div&#62;</code><br>R: <code>\1</code>
 </details>
 <details close>
 <summary>Clean and Code</summary><blockquote>
 <details close>
 <summary>Languages, Apparatus and Symbols</summary>
 - **lang-hbo**: Find instances of Hebrew<br>F: <code>(\[ְֱֲֳִֵֶַָֹֺֻּֽ֑֖֛֢֣֤֥֦֧֪֚֭֮֒֓֔֕֗֘֙֜֝֞֟֠֡֨֩֫֬֯־ֿ׀ׁׂ׃ׅׄ׆ׇאבגדהוזחטיךכלםמןנסעףפץצקרשתװױײ׳״]+-? ?)+)</code>
 
-- **lang-grc**: Find instances of Greek<br>F: <code>((?:\[\x{0300}-\x{036F}\x{0370}-\x{03FF}\x{1F00}-\x{1FFF}\x{20D0}-\x{20FF}\x{FE20}-\x{FE2F}]+\[,. ]&#42;)+)</code>
+- **lang-grc**: Find instances of Greek<br>F: <code>((?:\[\x{0300}-\x{036F}\x{0370}-\x{03FF}\x{1F00}-\x{1FFF}\x{20D0}-\x{20FF}\x{FE20}-\x{FE2F}]+\[,. ]\*)+)</code>
 
 - **lang-grc (2)**: Find instances of Greek<br>F: <code>(\[\p{Greek}]\[\p{Greek} ́¨ˆ̂˘̆̑̃ˋ̔̓ ͂.,’“;]+\b)</code>
 
 - **apparatus symbols**: Find apparatus symbols.<br>F: <code>(\[ℵ]|&#x(?:2135;|E(?:00\[021];|5(?:0\[45E6FA];|1\[034679];))))</code>
 
-- **check lang**: Find special `lang` characters<br>F: <code>&#60;span class=&#34;(\[^&#34;]+)&#34;&#62;(\[^A-Z]\[^<]&#42;\[āåâêëėèēîīôöòōûüū]\[^<]&#42;)&#60;/span&#62;</code>
+- **check lang**: Find special `lang` characters<br>F: <code>&#60;span class=&#34;(\[^&#34;]+)&#34;&#62;(\[^A-Z]\[^<]\*\[āåâêëėèēîīôöòōûüū]\[^<]\*)&#60;/span&#62;</code>
 
-- **extract lang**: Choose <mark>'Extract'</mark> to create a list of italicized words. Use this list to look for untagged lang or translit<br>F: <code>&#60;span class=&#34;(italic|i)&#34;&#62;(\[^<]&#42;)&#60;/span&#62;</code>
+- **extract lang**: Choose <mark>'Extract'</mark> to create a list of italicized words. Use this list to look for untagged lang or translit<br>F: <code>&#60;span class=&#34;(italic|i)&#34;&#62;(\[^<]\*)&#60;/span&#62;</code>
 
-- **ampersands**: replace ampersands<br>F: <code>(\[a-z]+\s&#42;)&(\s&#42;\[a-z]+)</code><br>R: <code>\1&#38;\2</code>
+- **ampersands**: replace ampersands<br>F: <code>(\[a-z]+\s\*)\*(\s\*\[a-z]+)</code><br>R: <code>\1&#38;\2</code>
```

Comment:
Looks like this is going to change the regex from `([a-z]+\s*)&(\s*[a-z]+)` to `([a-z]+\s*)*(\s*[a-z]+)`
Is that intended?

Reply:
That was not intended, I fixed it by putting an escaped ampersand in place of the asterisk.